libs/domain/site/src/services/currency/default-currency.service.ts

```diff
@@ -28,6 +38,22 @@ export class DefaultCurrencyService implements CurrencyService {
     );
   }
 
+  getCurrencySymbol(): Observable<string> {
+    return combineLatest([this.get(), this.localeService.get()]).pipe(
+      map(([currency, locale]) =>
+        (0)
+          .toLocaleString(locale, {
+            style: 'currency',
+            currency: currency,
+            minimumFractionDigits: 0,
+            maximumFractionDigits: 0,
+          })
+          .replace(/\d/g, '')
```

Comment:
Suggestion, just cut first char instead of using replace + Regex

Reply:
depends on locale position of the currency symbol can be various